q2_feature_table/tests/test_merge.py

```diff
@@ -167,6 +167,16 @@ def test_sum_overlapping_feature_ids(self):
                     ['S1', 'S2', 'S3', 'S4', 'S5', 'S6'])
         self.assertEqual(obs, exp)
 
+    def test_average(self):
```

Comment:
@BenKaehler, do you think we should add a test for a relative frequency table, too?

Reply:
Erm, at that level they’re all biom Tables. Should we build another test that calls it via the plugin interface?

Reply:
Could you just make a vanilla biom table with fractional values?

Reply:
I added a quick test for this.